doc/index.html

```diff
@@ -236,16 +236,16 @@
 			                    sudo usermod -G supergroup $username
 			                </p><br>
 			
-			                <h3>3.2 Get the code.</h3>
+			                <h3>3.2 Get the code</h3>
 			                Go to the home directory of the solution user in the node assigned for spot-setup and spot-ingest and clone the code:<br><br>
 			                <p class="terminal">
 			                    git clone https://github.com/apache/incubator-spot.git
 			                </p><br>
 			
-			                <h3>3.3 Edit apache spot (incubating) configuration.</h3>
+			                <h3>3.3 Edit apache spot (incubating) configuration</h3>
 			                Go to apache spot (incubating) configuration module to edit the solution configuration:<br><br>
 			                <p class="terminal">
-			                    cd /home/solution_user/incubator-spot/spot-setup<br>
+			                    cd /home/solution-user/incubator-spot/spot-setup<br>

```

Comment:
Wondering if there is a way in HTML to represent a placeholder. I think the original reason to use solution_user was that, feels like solution-user is the actual folder, just like incubator-spot. 
The main thing is, solution-user or solution_user is a placeholder for the actual user people choose to use.